happi/client.py

```diff
@@ -208,13 +195,11 @@ def create_device(self, device_cls, **kwargs):
             device = client.create_device(Device,   name='my_device' ...)
             device = client.create_device('Device', name='my_device',...)
 
-        See Also
-        --------
-        :attr:`.device_types`
         """
         # If specified by a string
-        if device_cls in self.device_types:
-            device_cls = self.device_types[device_cls]
+        if not inspect.isclass(device_cls) \
```

Comment:
Seems like you could get by without the `inspect.isclass` call

Reply:
That is true... the `device_cls in containers.registry` will take care if it is a string argument.
Addressed via https://github.com/pcdshub/happi/pull/126/commits/d71fabffde7ccadec073e2717023f704cfc82a71